config/puma.rb

```diff
@@ -4,7 +4,7 @@
 # the maximum value specified for Puma. Default is set to 5 threads for minimum
 # and maximum; this matches the default thread size of Active Record.
 #
-max_threads_count = ENV.fetch("RAILS_MAX_THREADS") { 5 }
+max_threads_count = ENV.fetch("RAILS_MAX_THREADS", 5)
 min_threads_count = ENV.fetch("RAILS_MIN_THREADS") { max_threads_count }
```

Comment:
j/w why the two ways of calling for max vs min?

Reply:
good point, I only changed this because rubocop started erroring on it but I'll change the min line as well to keep it consistent